addons/sys_attenuate/fnc_toggleDebugInfo.sqf

```diff
@@ -0,0 +1,129 @@
+#include "script_component.hpp"
+/*
+ * Author: ACRE2Team
+ * Gives the ability to view attenuation behavior at runtime to diagnose issues with compartment configs.
+ *
+ * Example:
+ * [ARGUMENTS] call acre_sys_attenuate_fnc_toggleDebugInfo;
+ *
+ * Public: No
+ */
+ 
+if (isNil {missionNameSpace getVariable "ACRE_SYS_ATTENUATE_DEBUG_DRAW"}) then {
+	ACRE_SYS_ATTENUATE_DEBUG_DRAW = {
+		private _speakers = allUnits;
+		private _vehicleSpeakers = (_speakers) select {_x in (crew (vehicle acre_player))};
+		private _outsideSpeakers = (_speakers - _vehicleSpeakers);
+
+		private _hintMsg = format ["<t align='left' size='1.15' color='#68bbff' font='PuristaMedium'>ACRE Attenuation</t><br/><br />"];
+
+		// Print player's vehicle role and compartment
+		if (!isNull objectParent acre_player) then {
+			_hintMsg = _hintMsg + format ["<t align='left' size='1.15' color='#68bbff' font='PuristaMedium'>Vehicle</t><br/>"];
```

Comment:
Instead of like
```sqf
_hintMsg = _hintMsg + format
```
I'd just do
```suggestion
			_hintMsg = format ["%1<t align='left' size='1.15' color='#68bbff' font='PuristaMedium'>Vehicle</t><br/>", _hintMsg];
```
Hell, https://community.bistudio.com/wiki/format implies you need to have `%1` at minimum, but it could be wrong (and I can't test it myself ATM).

Something like this could be faster, too (can't test ATM):
```sqf
//Create the var.
private _helpMsg = [];

//Do your stuff ...

_helpMsg pushBack "string";

//Do your stuff, including adding more strings ...

//Finally, show the hint, like done on line 110.
hintSilent parseText (_helpMsg joinString "");
```

Reply:
format doesn't require any arguments but you are correct in that I should not concatenate the string every time